rebasehelper/build_tools/rpmbuild_tool.py

```diff
@@ -63,8 +64,12 @@ def _build_rpm(cls, srpm, workdir, results_dir, builder_options=None):
                                                    env={'HOME': workdir},
                                                    output=output)
 
+        build_log_path = os.path.join(rpm_results_dir, 'build.log')
+
         if ret != 0:
```

Comment:
I think it would be nicer to use the same style of `ret` handling as in `MockBuildTool._build_rpm()`.